pyaedt/filtersolutions_core/export_to_aedt.py

```diff
@@ -0,0 +1,481 @@
+
+    @property
+    def polar_plot_enabled(self) -> bool:
+        """Flag indicating if the polar report format in the
+         exported filter to ``AEDT`` is enabled.
+
+        Returns
+        -------
+        bool
+        """
+        polar_plot_enabled = c_bool()
+        status = self._dll.getPolarPlot(byref(polar_plot_enabled))
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+        return bool(polar_plot_enabled.value)
+
+    @polar_plot_enabled.setter
+    def polar_plot_enabled(self, polar_plot_enabled: bool):
+        status = self._dll.setPolarPlot(polar_plot_enabled)
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+
+    @property
+    def table_data_enabled(self) -> bool:
+        """Flag indicating if the table data format in the
+         exported filter to ``AEDT`` is enabled.
+
+        Returns
+        -------
+        bool
+        """
+        table_data_enabled = c_bool()
+        status = self._dll.getTableData(byref(table_data_enabled))
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+        return bool(table_data_enabled.value)
+
+    @table_data_enabled.setter
+    def table_data_enabled(self, table_data_enabled: bool):
+        status = self._dll.setTableData(table_data_enabled)
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+
+    @property
+    def optimitrics_enabled(self) -> bool:
+        """Flag indicating if the optimitric parameters in the
+         exported filter to ``AEDT`` is enabled.
+
+        Returns
+        -------
+        bool
+        """
+        optimitrics_enabled = c_bool()
+        status = self._dll.getOptimetrics(byref(optimitrics_enabled))
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+        return bool(optimitrics_enabled.value)
+
+    @optimitrics_enabled.setter
+    def optimitrics_enabled(self, optimitrics_enabled: bool):
+        status = self._dll.setOptimetrics(optimitrics_enabled)
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+
+    @property
+    def optimize_after_export_enabled(self) -> bool:
+        """Flag indicating if the optimization option after exporting to ``AEDT`` is enabled.
+
+        Returns
+        -------
+        bool
+        """
+        optimize_after_export_enabled = c_bool()
+        status = self._dll.getOptimizeAfterExport(byref(optimize_after_export_enabled))
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+        return bool(optimize_after_export_enabled.value)
+
+    @optimize_after_export_enabled.setter
+    def optimize_after_export_enabled(self, optimize_after_export_enabled: bool):
+        status = self._dll.setOptimizeAfterExport(optimize_after_export_enabled)
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+
+    def append_design_to_aedt(self, export_format: ExportFormat.DIRECT):
```

Comment:
Would it be easier to find these if there is one `export()` function that takes and option for overwrite/append? This module is called export, most places in the docstrings we mention 'export', but neither function that does the export has export in the name.

Is there a function to export the script? I didn't see it here. I might have missed it. It might make sense to include the script export as one of the options for the combined `export()` function.